spark/src/main/scala/org/apache/spark/sql/delta/catalog/DeltaCatalog.scala

```diff
@@ -523,6 +532,27 @@ class DeltaCatalog extends DelegatingCatalogExtension
     }
   }
 
+  private def getTablePropsAndWriteOptions(properties: util.Map[String, String])
```

Comment:
This is the same code moved from https://github.com/delta-io/delta/pull/3674/files#diff-e880b2b407605f237c5bb566e3e06f981a4fbbb6ff675d7832b2cf5d2daa6b3dL547